bundles/org.openhab.ui/web/src/components/config/editor/hint-rules.js

```diff
@@ -144,9 +149,18 @@ function buildModuleStructure (cm, moduleType) {
   return ret
 }
 
+function hintSceneItems (cm, line, parentNr) {
```

Comment:
What I have noted during testing this, is that code hinting does not open up if you add a new line to the end of the section. I know how to fix this, but let's discuss this in another PR since this may have some side effects.

I also wondered if we could provide the user with some information how to "force" hinting (Ctrl + Space).